updating links (#210)
rstudio/education.rstudio.com: content/learn/intermediate/index.Rmd

Once you've gotten started learning R, you can expand your skills by exploring many of the specialized capabilities of R. Here are 6 of the most common areas that people who already have some experience in R find particularly rewarding to learn.

- __Grab some cheat sheets.__ No one can possibly remember all the functions and arguments for every R package, which is why cheat sheets were invented. Posit publishes a [free collection of cheat sheets](https://opensource.posit.co/resources/cheatsheets/) for the most popular R features and packages to help jog your memory. If you decide you'd like to collect them all, you may clone [the cheat sheet github repository](https://opensource.posit.co/resources/cheatsheets/). 

> - __Learn to get help.__ Everyone gets stuck. Learning where and how to ask for R help is a powerful skill to hone. The Tidyverse site offers some expert advice for [how to help others help you](https://www.tidyverse.org/help/). One package you'll grow to love is the [reprex package](https://reprex.tidyverse.org) for creating reproducible R code examples. Read through the [reprex](https://reprex.tidyverse.org) articles, which feature loads of animated gifs to illustrate the steps like [_Magic reprex_](https://reprex.tidyverse.org/articles/articles/magic-reprex.html) and [_Using datapasta with reprex_](https://reprex.tidyverse.org/articles/articles/datapasta-reprex.html). Where to ask for help? The [Posit Community](https://forum.posit.co/) is a warm and welcoming online discussion forum to ask (and answer!) any questions about using R.

Reviewer: ```suggestion
- __Learn to get help.__ Everyone gets stuck. Learning where and how to ask for R help is a powerful skill to hone. The Tidyverse site offers some expert advice for [how to help others help you](https://www.tidyverse.org/help/). One package you'll grow to love is the [reprex package](https://reprex.tidyverse.org) for creating reproducible R code examples. Read through the [reprex](https://reprex.tidyverse.org) articles, which feature loads of animated gifs to illustrate the steps like [_Magic reprex_](https://reprex.tidyverse.org/articles/articles/magic-reprex.html) and [_Using datapasta with reprex_](https://reprex.tidyverse.org/articles/articles/datapasta-reprex.html). Where to ask for help? The [Posit Community](https://forum.posit.co/) is a warm and welcoming online discussion forum for asking (and answering!) any questions about using R.
```

-	__Simplify your model explorations with *tidymodels*.__ Much of data science involves modeling, but each modeling package seems to invent its own interface and arguments. Enter [tidymodels](https://github.com/tidymodels), a meta-package for modeling and analysis that shares the underlying design philosophy, grammar, and data structures of the tidyverse. If you previously have used caret for a uniform modeling interface, the [tidymodels package parsnip](https://tidymodels.github.io/parsnip/) is its more up-to-date child. While this project is still under development, it promises to dramatically simplify model exploration. RStudio's [Edgar Ruiz](https://twitter.com/theotheredgar) wrote up [_A Gentle Introduction to tidymodels_](https://rviews.rstudio.com/2019/06/19/a-gentle-intro-to-tidymodels/) to get you started.

> - __Explore other specialized packages.__ R attracts data scientists because of its more than 13,000 packages that address nearly every use case. If you're interested in genomics, you'll want to spend some time learning the [*bioconductor* collection of packages](https://bioconductor.org/). If you're working with Big Data on Spark clusters, check out [sparklyr](https://spark.posit.co). If you want to dive into finance, you'll probably want to start with [quantmod](http://www.quantmod.com). To find out what packages you should explore, we recommend some of the topic-based package catalogs such as [Awesome R](https://awesome-r.com/) or the [CRAN task views](https://cran.r-project.org/web/views/).

Reviewer: ```suggestion
- __Explore other specialized packages.__ R attracts data scientists because of its more than 13,000 packages that address nearly every use case. If you're interested in genomics, you'll want to spend some time learning the [*bioconductor* collection of packages](https://bioconductor.org/). If you're working with Big Data on Spark clusters, check out [sparklyr](https://spark.posit.co). If you want to dive into finance, you'll probably want to start with [quantmod](http://www.quantmod.com). To find out which packages to explore, we recommend topic-based package catalogs such as [Awesome R](https://awesome-r.com/) or the [CRAN task views](https://cran.r-project.org/web/views/).
```
